peachjam/tasks.py

```diff
@@ -206,6 +206,19 @@ def convert_source_file_to_pdf(source_file_id):
     log.info("Conversion to PDF done")
 
 
+@background(queue="peachjam", remove_existing_tasks=True)
+def convert_html_to_pdf(judgment_id):
+    from peachjam.models import Judgment
```

Comment:
I think this should work for any file, not just judgment.

Reply:
done